playground/backend/internal/environment/environment_service_test.go

```diff
@@ -113,13 +120,13 @@ func Test_getSdkEnvsFromOsEnvs(t *testing.T) {
 		},
 		{
 			name:      "default beam envs",
```

Comment:
Please add a comment for this test case.

Reply:
We haven't used comments when were writing those tests. It doesn't make sense to comment only one test instead of all. Please add new task to comment all